### PR TITLE
docs(autok3s): update alibaba CN and EN

### PR DESCRIPTION
**Pull request by yzeng25**
- delete redundant description on README.md, in CN and EN.
- add "简体中文/English" tags in the beginning of alibaba CN and EN docs.
- refactor alibaba CN and EN docs.

### CODE REVIEW

File: docs/i18n/zh_cn/alibaba/README.md [hunk 1]
```diff
@@ -57,73 +65,46 @@ export ECS_ACCESS_KEY_SECRET='<secret-access>'
...
       "Effect": "Allow"
     }
   ]
 }
 ```
 
 ### 设置安全组
-ECS实例需要应用以下安全组规则:
+
+ECS 实例需要应用以下安全组规则：
```
Comment (JacieChao) [inline docs/i18n/zh_cn/alibaba/README.md:107]: ECS实例 **至少** 需要应用以下安全组规则

File: docs/i18n/zh_cn/alibaba/README.md [hunk 2]
```diff
@@ -136,57 +117,88 @@ InBound     TCP         2379,2380 K3s server nodes   (Optional) Required only fo
 OutBound    ALL         ALL       ALL                Allow All
 ```
 
-## 使用方式
-更多参数请运行`autok3s <sub-command> --provider alibaba --help`命令。
+## 创建集群
+
+请使用`autok3s create`命令在 VM 实例中创建集群。
```
Comment (JacieChao) [inline docs/i18n/zh_cn/alibaba/README.md:122]: 在阿里云 ECS 实例中

File: docs/i18n/zh_cn/alibaba/README.md (hunk 1 above)
Comment (yzeng25) [reply docs/i18n/zh_cn/alibaba/README.md]: fixed

File: docs/i18n/zh_cn/alibaba/README.md (hunk 2 above)
Comment (yzeng25) [reply docs/i18n/zh_cn/alibaba/README.md]: fixed

**Pull request merged**